frontend/src/components/StackedGroupedBar/index.jsx

```diff
@@ -37,8 +47,9 @@ const StackedGroupedBar = ({ data, indexKey, groupKeys, stackKeys }) => {
         }}
         groupMode="grouped"
       />
-    <div className="custom_tooltip" ref={customToolTipRef}></div>
-  </>;
+      <div className="custom_tooltip" ref={customToolTipRef}></div>
+    </>
+  );
```

Comment:
Looks like your editor has different code style settings and therefore these files were changed. Normally would be nice to leave out any irrelevant changes from PRs, but to move forward smoothly these are ok.